peachjam_search/documents.py

```diff
@@ -53,7 +53,7 @@ class SearchableDocument(Document):
     order_outcome = fields.KeywordField(attr="order_outcome.name")
 
     # GenericDocument, LegalInstrument
-    author = fields.KeywordField(attr="author.name")
+    authors = fields.KeywordField(attr="authors.name")
```

Comment:
This won't work because `document.authors.name` doesn't work. look instead at how we handle it for alternative names, see `prepare_alternative_names`